src/unitxt/operators.py

```diff
@@ -671,6 +672,69 @@ def process(self, stream: Stream, stream_name: str = None) -> Generator:
                 yield instance
 
 
+class FilterByCondsOnValues(SingleStreamOperator):
+    """
+    Filters a stream, yielding only instances that match conditions specified as stringed-lambda on values in the provided fields.
+
+    Args:
+        values (Dict[str, Any]): For each field, the values that instances should match to be included in the output.
+    """
+
+    required_values: Dict[str, Any] = {}
+
+    def process(self, stream: Stream, stream_name: str = None) -> Generator:
+        for instance in stream:
+            filter = False
+            for key, value in self.required_values.items():
+                filtlambda = eval(value)
+                if not key in instance:
+                    raise ValueError(
+                        f"Required filter field ('{key}') in FilterByCondsOnValues is not found in {instance}"
+                    )
+                if not filtlambda(instance[key]):
+                    filter = True
+            if not filter:
+                yield instance
+
+
+class FindAndStoreMostCommonValues(MultiStreamOperator):
+    field_name: str
+    stream_name: Optional[str] = "train"
+    top_percent: Optional[int] = 80
+    min_relative_freq_percent: Optional[int] = 5
+    new_field: Optional[str] = None
+    """
+    Given a MultiStream, a stream_name, and a field name, and two margins of commonality.
+    Do nothing (exception thrown) if the multistream does not contain a stream named stream_name, or if the multistream does not
+    contain a field (feature) named field_name.
+    Count the frequency of each value, over the stream named stream_name. Sort the values by frequency.
+    Discard each value that is not at the top_percent of that sorted list.
+    Discard each value whose frequency, expressed as percent of the number of rows in the stream, is below min_relative_freq_perent
+    Then, for all streams of the multistream, for all rows of each stream, add a new field, named
+    'new_field' whose value will be the sored list of the remaining values.
+    (that is each and every row in the multistream will carry that very same field-value pair.)
+    """
+
+    def process(self, multi_stream: MultiStream) -> MultiStream:
+        stream = multi_stream[self.stream_name]
+        all_values = []
+        for instance in stream:
+            all_values.append(instance[self.field_name])
```

Comment:
Need to support list elements

Reply:
As I wrote above: not sure if to support them individually: count the frequency of each individual, and then which records are to stay? those whose lists consist of all frequent values? whose lists contain at least one frequent value? 
Or rather: measure frequencies of whole lists, and then the same rule used for individual values would apply for whole lists, and the issue is just to allow, programmatically, lists of str, and not only   str.